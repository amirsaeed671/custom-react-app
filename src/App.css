* {
    font-family: 'Roboto', sans-serif;
}

html, body, #root {
    height: 100%;
    width: 100%;
    margin: 0;
    padding:0;
}

label {
    font-size: 0.8em;
    margin-bottom: 0.4em;
}