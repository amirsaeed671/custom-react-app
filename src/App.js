import React from 'react';
import './App.css';

function App() {
  return <h1 data-testid="heading">Hello World</h1>;
}

export default App;
