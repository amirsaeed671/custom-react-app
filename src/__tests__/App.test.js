import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

test('should render hello', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
