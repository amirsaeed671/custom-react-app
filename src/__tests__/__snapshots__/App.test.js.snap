// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render hello 1`] = `
<DocumentFragment>
  <h1>
    Hello World
  </h1>
</DocumentFragment>
`;
